tests/conftest.py

```diff
@@ -97,34 +95,20 @@ def notify_db(notify_api):
     db.session.remove()
     db.engine.dispose()
 
-    # TODO - Properly delete the database after all workers finish?
+    # TODO - We should delete the test database after all tests finish, but how do we do this
```

Comment:
since you've touched this comment anyway, can you please add API-900 after `TODO - ` :)